MarkSkalskyi/README.md

```diff
@@ -0,0 +1,42 @@
+# Getting started with [HDRezka](https://rezka.ag)
+
+HDRezka is a website that allows you to watch your favorite movies and series. If you need a guide for signing in, you're in the right place.
+
+**Movies? HDRezka!**
+
+## Step 1: Registration
+
+1. Firstly, visit the HDRezka website at [HDRezka](https://rezka.ag).
+   This is how it looks like:
+   ![Screenshot of the HDRezka website](https://ibb.co/JxMxSZ1)
+
+2. To register, click the button labeled **"Регистрация"** (Registration).
+   ![Register Button](https://ibb.co/ScJ26s2)
+
+3. A registration window will pop up. Enter your *Login*, *E-Mail*, and *Password*.
+   ![Registration Window](https://ibb.co/qjV8S82)
+
+4. Click the big blue button labeled "РЕГИСТРАЦИЯ" (Registration).
+
+5. Hurray! You are now registered and can start watching all the movies available on the site.
+
+## Step 2: Searching
+
+1. Locate the search bar, which is placed under the "Профиль" (Profile) button. Previously, it was the "Регистрация" (Registration) button.
+   ![Search Bar](https://ibb.co/vPZyqSN)
+
+2. Now you can search for movies and series. Let's try searching for `Titanic` for example.
+
+3. This is how the search results will look like:
+   ![Results: Titanic](https://ibb.co/88m17jD)
+
+## Step 3: Final step
+
+Now that you've found the movie you were looking for, you need to choose a translation to your liking and start watching the movie.
```

Comment:
Don't use you need to choose. Instead, use just choose:
Now that you've found the movie you were looking for, choose a translation to your liking and start watching the movie.